src/views/components/global/Toolbar/index.vue

```diff
@@ -44,13 +47,37 @@ export default Vue.component(name, {
 </script>
 
 <style lang="scss" scoped>
+  @import "~vue-material/dist/theme/engine";
```

Comment:
Why is the import in both?

Reply:
because one of these style tags is scoped and the other is not. it didn't work when i didn't have this import here